src/renderer/viz/expressions/classification/ViewportQuantiles.js

```diff
@@ -68,8 +93,23 @@ export default class ViewportQuantiles extends Classifier {
                 }
             }
             const percentileValue = i / histogramBuckets * (max - min) + min;
-            brs.push(percentileValue);
             breakpoint.expr = percentileValue;
         });
     }
+
+    _getAccumHistogramFrom (histogram) {
+        let prev = 0;
+        const accumHistogram = histogram.map(({ y }) => {
+            prev += y;
+            return prev;
+        });
+        return accumHistogram;
```

Comment:
Why not "reduce"?

```
return histogram.reduce((acc, { y }) => {
    return acc + y;
}, 0);
```

Reply:
I had a look, and the map preserves the original array disposition, which is needed. Using reduce here would require accumulating but also returning the array on every iteration, so IMO map is nice in this particular case.

Reply:
Got it! Thanks for the offline explanation. 😅